### ISSUE TITLE
Few enchancements to add for the next release

### ISSUE THREAD

**Issue by TearJoy** [Futur3Sn0w/PaperProton#3]
Hi there, first and foremost thanks for the userChrome. I love MaterialFox and having it revived is wonderful.

That said, after a few minutes using it, I noticed something... off. I've been using the Lepton userChrome ever since MaterialFox has been out of date. I loved it for being more compact than Proton, toolbars, context menus, etc. Using PaperProton, I noticed that the bookmarks bar is a bit too chonky, which triggered RFP letterboxing to kick in. Second is that the context menu remains unchanged and is still eating much space. 
Using PaperProton, The ETP shield is just gone. I would like the shield to come back 
Some Firefox native things are also missing (idk if it's intentional or not). One of the things I've mentioned being gone is ETP. Using PaperProton, The ETP shield is just gone. I would like the shield to come back in the next release if that isn't too much to ask, or implement it different like how MaterialFox does it (Will add more enchancements as I use it)

With Lepton: 
![image](https://user-images.githubusercontent.com/81557214/124300307-f687ed00-db4d-11eb-9b6f-0f78cc8e7a11.png)

PaperProton:
![image](https://user-images.githubusercontent.com/81557214/124300377-099abd00-db4e-11eb-8960-64b3039a6544.png)
(White Bars are RFP Letterboxing)

A bug I've noticed with PaperProton:
![image](https://user-images.githubusercontent.com/81557214/124301109-e0c6f780-db4e-11eb-83d0-e853dd829e86.png)
MaterialFox:
![image](https://user-images.githubusercontent.com/81557214/124301757-c5102100-db4f-11eb-8638-bcdd65525009.png)

tldr my recommendations for the next release:

- More narrow Bookmarks bar
- More compact context menus, icons are optional
- Add ETP shield icon back
- To be added

How MaterialFox implements the ETP shield:
[Streamable Link](https://streamable.com/4ripww)
